code/modules/lore_codex/news_data/main.dm

```diff
@@ -339,4 +344,60 @@
+
+/datum/lore/codex/page/article32
+	name = "01/25/63 - Moravec Nephew Announces Vir Governor Candidacy"
+	data = "The Sol Economic Organization has announced that Calvert Moravec, nephew of NanoTrasen CEO Albary Moravec will be running under their ticket in the upcoming Vir gubernatorial elections. Calvert has stated that he will run on a pro-business platform, and has chosen Vir to do so due to the 'Unique beauty and economic prospects of an interstellar crossroads such as Vir'.\
+	<br><br>\
+	Despite being a lifelong resident of Alpha Centauri, Moravec was recently approved for Sif citizenship, making him eligible for local candidacy and has reportedly moved into a luxurious New Reykjavik penthouse. Perhaps best known for his soaring stock market investment success over the previous few years, Calvert's first foray into politics is not wholly unexpected as the Moravec family has long leveraged their wealth in international affairs, though successful election would mark their first sitting member of the Colonial Assembly. Fellow SEO candidate Councillor Hal Wekstrom has expressed his full support for Moravec.\
+	<br><br>\
+	Three candidates will be elected as representatives to the Colonial Assembly later this year, with the most popular also attaining the position of system governor."
+
+/datum/lore/codex/page/article33
+	name = "01/27/63 - Vani Jee Orbital Tour Cut Short"
+	data = "Icarus Front Representative Candidate Vani Jee has delayed the remainder of her campaign tour of orbital colonies and outposts around the Vir system after an alleged altercation with NanoTrasen security.\
+	Candidate Jee had been visiting the NLS Southern Cross, a NanoTrasen station in Sif orbit to recieve a corporate tour and meet with voters, when her concluding question and answer session was interrupted by hecklers, leading to the event being cut short. Jee alleges that footage of the event was seized by NanoTrasen corporate security and has accused the trans-stellar corporation of the intentional intimidation of Icarus Front and Sol Economic Organization candidates in what she describes as 'a clear display of corruption in favour of company-favourite Calvert Moravec', though she has praised the individual employees of the Cross for their hospitality and thought-provoking questions.\
```

Comment:
`receive` a corporate tour

Reply:
Also wouldn't they be favoring SEO candidates over Shadow Coalition canditions, given that SEO is basically TSC's: the party?

Reply:
Ah you're right that should be SC not SEO (It's specifically referring to the candidate who is literally related to the NT CEO)